Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/computers_shipyard.yml

```
@@ -221,3 +221,27 @@
       state: shipyard_blackmarket
     - map: ["computerLayerKeys"]
       state: blackmarket_key
+
+- type: entity
+  id: ComputerShipyardSr
+  parent: [BaseStructureDisableToolUse, BaseStructureIndestructible, ComputerShipyard]
+  name: sr shipyard console
+  description: Used to buy vessels exclusive to the station representative
+  components:
+  - type: ActivatableUI
+    key: enum.ShipyardConsoleUiKey.Sr
+  - type: UserInterface
+    interfaces:
+      enum.ShipyardConsoleUiKey.Sr:
+        type: ShipyardConsoleBoundUserInterface
+  - type: Sprite
+    sprite: _NF/Structures/Machines/computers.rsi
+    layers:
+    - map: ["computerLayerBody"]
+      state: computer_sr
+    - map: ["computerLayerKeyboard"]
+      state: generic_keyboard
+    - map: ["computerLayerScreen"]
+      state: shipyard_sr
```

shipyard_sr should be unshaded, it's a brightly lit computer console.
```suggestion
      state: shipyard_sr
      shader: unshaded
```

---

This seems to be applied automatically.  My mistake.

---

It _is_ unshaded. `ComputerShipyardBase` defines a visual for this layer that makes it unshaded when the computer is powered, and hides it when unpowered.